_replication/replication-methods/log-based-incremental-replication.md

```diff
@@ -0,0 +1,260 @@
+
+          Because of this and [how extraction is performed using {{ page.title }}](#how-log-based-incremental-replication-works), changes in structure will cause extraction errors.
+
+          For example, consider this table:
+
+          | id | name      | type     |
+          |----+-----------+----------|
+          | 1  | Finn      | human    |
+          | 2  | Jake      | dog      |
+          | 3  | Bubblegum | princess |
+
+          The values for these fields might look like this in a binary log:
+
+          ```
+          1,Finn,human,2,Jake,dog,3,Bubblegum,princess
+          ```
+
+          Stitch's MySQL and PostgreSQL integrations use JSON schema validation to ensure that values in log messages are attributed to the correct fields when data is loaded into your destination. For this reason, schema changes in a source - whether it's changing a column's data type or re-ordering columns - will cause an extraction error to occur.
+
+          As {{ page.title }} is a seamless process, an extraction error for any one table in a replication job will disrupt replication from any other tables using {{ page.title }}.
+
+          If the column order or data types of a source table change in any capacity, the integration will not persist new or updated records that use this updated schema, as it does not have a means of attributing values to their proper fields based on the ordinal set when compared to the expected schema that was previously detected.
+
+          Let's look at an example. Here's the same table from before, now with a new `age` column:
+
+          | id | name      | age | type     |
+          |----+-----------+-----+----------|
+          | 1  | Finn      | 15  | human    |
+          | 2  | Jake      | 9   | dog      |
+          | 3  | Bubblegum | 19  | princess |
+
+          And the values in the binary log:
+
+          ```
+          1,Finn,15,human,2,Jake,9,dog,3,Bubblegum,19,princess
+          ```
+
+          Where Stitch previously detected three columns, the log messages now contain data for four columns. Because the log messages don't contain field information and are read in order, Stitch would be unable to determine what column the `15`, `9`, and `19` values are for.
+
+      - title: "Limitation 5: Frequent updates can increase row usage"
+        anchor: "limitation-5--row-usage"
+        content: |
+          While {{ page.title }} is the most accurate way to replicate your data, it has the potential to consume a large amount of your row quota. To factors can contribute to this:
+
+          1. Each modification to a record may a single log message for the modification, and
+          2. Stitch reads log messages in order during extraction
+
+          This means that each log message Stitch extracts counts as a replicated row, even if all the messages are for the same record.
```

Comment:
if multiple updates are loaded within the same batch, they should not actually be counted as multiple records, though this would add a level of complexity to the explanation that I'm not sure is worth getting into?

Reply:
I'd thought they *would* be counted towards usage, which is why the section is in here. Is that not the case? If it isn't, and this sort of scenario won't affect the total number of rows, I think this can come out.